ote/src/cljs/ote/views/route/trips.cljs

```diff
@@ -136,20 +138,25 @@
 (defn trips-list [e! route]
   (let [stop-sequence (::transit/stops route)
         stop-count (count stop-sequence)
-        trips (::transit/trips route)]
-
+        trips (::transit/trips route)
+        empty-calendar? (empty? (first (:ote.db.transit/service-calendars route)))]
     [:div.route-times
      [:table {:style {:text-align "center"}}
       [route-times-header stop-sequence]
       [:tbody
        (doall (map-indexed (partial trip-row e! stop-count) trips))]]
+
+     (when empty-calendar?
```

Comment:
This is an opinion, but wouldn't it be better to keep the infobox always visible? For example, I forgot the instructions after I opened the calendar open and went back to trips editor.

Reply:
I'll leave this as it is now based on UI specs but we can remove it later if needed.